### ISSUE TITLE
Parsing bug in avisynth code

### ISSUE THREAD

**Issue by mirkosp** [pinterf/xy-VSFilter#1]
When trying to hardsub scripts with textsub in avisynth, the last line of sub will be ignored if there isn't a new empty line below the last one.

**Comment by pinterf** [comment]
Good catch. Fixed in commit https://github.com/pinterf/xy-VSFilter/commit/569cc5aa2f9c658de6ea52ad8c2c7414fc5072f0
